src/modules/Rotate/Rotate.js

```diff
@@ -0,0 +1,46 @@
+module.exports = function Rotate(pixels, pixels2, options, rotate_value, width, height, cos, sin){
+  var imagejs = require('imagejs');
+  var height_half = Math.floor(height / 2);
+  var width_half = Math.floor(width / 2);
+  var dimension = width + height;
+
+  if (rotate_value % 360 == 0)
+    return pixels;
+  function copyPixel(x1, y1, x2, y2){
```

Comment:
Hi, couldn't this get extra `pixels1, pixels2` parameters and then we'd have only one function instead of 2?

Reply:
ok, would do the changes to make a 1 function.

Reply:
@jywarren any other changes that are required??

Reply:
@jywarren Had done the changes to make a single function instead of 2